module1.js

```diff
@@ -23,179 +23,106 @@ function question1(name1) {
   }
 }
 
-
 /**
-  +++++++++++++++++++++++++++++++++++++++++++
   Question 2:
-  IF / ELSE Statements:
-  Check if two numbers are equal, if they are return true or otherwise return false
-  @param {number} number1 a regular integer
-  @param {number} number2 a regular integer
-  @returns {boolean} true or false
-*/
+  Write a function that compares two input integers.
 
-// function question2(number1, number2) {
-//   if (number1) {
-//     // TODO:
-//     return;
-//   } else {
-//     // TODO:
-//     return;
-//   }
-// }
+  Return true if they are equal, or false otherwise.
+*/
 
+function question2(number1, number2) {
+  // TODO:
+}
 
 /**
-  +++++++++++++++++++++++++++++++++++++++++++
   Question 3:
-  Is the number even or odd?
-  @param {number} number1 a regular integer
-  @returns {string} odd or even
-*/
+  Write a function that checks if an input integer is an even or an odd number.
 
-// function question3(number1) {
-//   if (number1) {
-//     return;
-//   }
-// }
+  Return a string "odd" if the number is odd, or a string "even" if the number is even.
+*/
 
+function question3(number) {
+  // TODO:
+}
 
 /**
-  +++++++++++++++++++++++++++++++++++++++++++
   Question 4:
-  Write a for loop that loops over the numbers from 0 to 100, returning an array containing only every second number. So it should return [ 0, 2, 4, 6, 8, 10 .... etc.]
+  Write a function that takes an array of integers as input, and returns an array containing only every second element.
 
-  @param none
-  @returns [array 0 ... 100] 
+  Use the .forEach() or .map() functions to do this.
```

Comment:
Hmm, not that I look at this again, `.filter` would be the best choice. Not sure how to word this without directly mentioning that. Maybe that's not a bad thing?

Reply:
We could refer to "iteration methods" in general: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#Iteration_methods

Reply:
Done